e2e/applied-rule-removed-node-with-cache/rector.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Caching\ValueObject\Storage\FileCacheStorage;
+use Rector\Config\RectorConfig;
+use Rector\DeadCode\Rector\ClassMethod\RemoveEmptyClassMethodRector;
+use Rector\DeadCode\Rector\If_\RemoveAlwaysTrueIfConditionRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->cacheClass(FileCacheStorage::class);
```

Comment:
This is the added line to the config from the original e2e/applied-rule-removed-node folder.
It makes the cache persistent between job steps